src/main/java/seedu/address/logic/parser/ArgumentMultimap.java

```diff
@@ -50,6 +50,17 @@ public Optional<String> getValue(Prefix prefix) {
         return values.isEmpty() ? Optional.empty() : Optional.of(values.get(values.size() - 1));
     }
 
+    /**
+     * Checks if the {@code prefix} only occurs once in the user input.
+     *
+     * @param prefix the prefix to check against
+     * @return true if the {@code prefix} only occurs once, otherwise false
+     */
+    public boolean isUniquePrefix(Prefix prefix) {
```

Comment:
I understand that this method is used to ensure user only inputs unique prefixes (i.e. no cases such as `delete p/87654321 p/12345678`), and is called [here](https://github.com/AY2223S1-CS2103T-W09-1/tp/pull/84/files#:~:text=argMultimap.isUniquePrefix(PREFIX_PHONE)), but would it be possible to abstract this out into the [`getValue()` method](https://github.com/AY2223S1-CS2103T-W09-1/tp/pull/84/files#:~:text=public%20Optional%3CString%3E%20getValue(Prefix%20prefix)) directly?